atomate/atat/fireworks/core.py

```diff
@@ -131,7 +131,7 @@ def __init__(self, disordered_struct,
         get_bestsqs_cmd = "mcsqs -best"
 
         # write the mcsqs version to a file for provenance
-        write_version_cmd = "mcsqs -v | head -n 1 > mcsqs_version.txt"
+        write_version_cmd = "mcsqs -v &> mcsqs_version1.txt; head -n 1 mcsqs_version1.txt > mcsqs_version.txt; rm -f mcsqs_version1.txt"
```

Comment:
Can you explain what this is trying to do?

Reply:
Looks like `mcsqs -v` writes to stderr: a correct version could be `mcsqs -v 2>&1 | head -n 1 > mcsqs_version.txt` ... there's no need to write to file, read the file, and then delete the file again, that's what pipes are for!

Reply:
Okay, sounds good! Yeah, that was my stupid and fast workaround...